wolfssl/wolfcrypt/aes.h

```diff
@@ -398,7 +399,8 @@ struct Aes {
 
 #ifdef WOLFSSL_AES_XTS
 typedef struct XtsAes {
-    Aes aes;
+    Aes aes_encrypt;
+    Aes aes_decrypt;
```

Comment:
Don't want this decrypt if it can be helped.
At least put it in a #ifdef as normal code won't use this.

Reply:
fixed.